pkg/asset/agent/image/oc.go

```diff
@@ -288,6 +295,52 @@ func (r *release) verifyCacheFile(image, file, architecture string) (bool, error
 	return false, nil
 }
 
+// Remove any existing files in the cache.
+func removeCacheFile(path string) error {
+	var err error
+	if strings.HasSuffix(path, "*") {
+		err = filepath.Walk(filepath.Dir(path), func(file string, info fs.FileInfo, err error) error {
```

Comment:
nit: I think we could also use `filepath.Glob(path)` for this, which might save a bit of code.

Reply:
Yeah, using Glob is a bit cleaner, changed this function and getExtractedFileFromWildcard()